### PR TITLE
Silent some warnings

### PR DESCRIPTION
**Pull request by UK992**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/10466)

### CODE REVIEW

**Comment by KiChjang** [comment]
@bors-servo r+

Thanks!

File: components/layout/display_list_builder.rs [hunk 1]
```diff
@@ -1440,7 +1440,7 @@ impl FragmentDisplayListBuilding for Fragment {
     fn adjust_clipping_region_for_children(&self,
                                            current_clip: &mut ClippingRegion,
                                            stacking_relative_border_box: &Rect<Au>,
-                                           is_absolutely_positioned: bool) {
+                                           _is_absolutely_positioned: bool) {
```
Comment (jdm) [inline components/layout/display_list_builder.rs]: If we're not using this argument, it's either a bug or we should remove it :)

cc @mbrubeck

**Comment by jdm** [comment]
@bors-servo: r-
Until we deal with the argument one way or another.

File: components/layout/display_list_builder.rs (hunk 1 above)
Comment (jdm) [inline components/layout/display_list_builder.rs]: Sorry, I really mean cc @pcwalton who removed the uses of this argument in 1e884ddc6969989fd2837d5262ff8cceb0ca2f9f.

Comment (mbrubeck) [inline components/layout/display_list_builder.rs]: Oops, I submitted my own fix for this in #10469 before I saw this PR.  Sorry!  If that lands, then this commit can be dropped.

**Comment by UK992** [comment]
Commit for silent warning dropped.

**Comment by KiChjang** [comment]
@bors-servo r+

Thanks!

**Pull request merged**